### PR TITLE
Feature/travis pypi

### PR DESCRIPTION
**Pull request by Richard-Mathie**
Pull Request to see if this triggers the travis build

### CODE REVIEW

**Comment by Richard-Mathie** [comment]
it builds but
```
Skipping a deployment with the pypi provider because the current build is a pull request.
```

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
+- python setup.py -q install
 script: py.test
+deploy:
+  provider: pypi
+  user: RichardMathie
+  password:
+    secure: nhSqOL/W9Q74AGPMAq8s7ItCVuJh7nSx4XiKAcLB9qlG4trBMGFOWOC9LH4EvI/luEqEKz6DU/HuUxEx9CPNV75ld3p9vggUvrPk9RC2rL71XhqqDotcLNo2chbDUHgrBwzHBr88NkCejlx4GVah/i6egYOJFHK5fZqqqQaOPGmY4cQISQ33E9Kyh8dP4uYzLaDWp/TTSVkhMl/micidfOWIVAJW4vtVGupbcfIXc6QhpXWI6RNbVa9UfKEijdu2pf+p5KJSQuQz6L86bJjklZVLqGGguLdX/omdA1sF+SrrwAVwNpOTlRj+30vrVUPIk/Ifhvw20dwvHiTYW4GL38OBRicU9+rhEaEC1swbgutNoqcJgmCD8HTB3mG3hKSIkK8vSHcUZCwjVfmMhRGF4HMINi7eT1PyQIxQZsGvtQFup5vgbq2TutA/JyVEBUqx6duSk0atpkhuqo4rPT8P2SXTmWjL9jDjApY7wGRTc2EbWYD3/Ycjesi2BGkhnLqNHwL+Sa5PflNq6+BaMPxyKznvL7vaf/+kzbMQewc8JdV3ad84wkoUK5dO31o/gMqhdLdrU226B7Doj933s+ev0jT61RmAtTqtiLSYlXjbCg1WkKqbsCUuyTGRMxth57gqLP/hzdxAg386nWaSGt3iRnkTRRkFVF0h0wPqBlJzjoM=
+  on:
+    # tags: true
+    distributions: sdist bdist_wheel bdist_egg
```
Comment (thegeorgeous) [inline .travis.yml:31]: do we need a source distribution?

**Comment by thegeorgeous** [comment]
I guess the `tags: true` will work only if it is on the master branch.

File: .travis.yml (hunk 1 above)
Comment (Richard-Mathie) [reply .travis.yml:31]: Well there is nothing compiled in this module, it is just 1 python file, so source dist is probably the minimal thing to do?

**Comment by Richard-Mathie** [comment]
Not sure. But Travis CI was only building on master or pull request.

>5. `tags`: When set to `true`, the application is deployed when a tag is applied to the commit. This causes the `branch` condition to be ignored.

https://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A

File: .travis.yml (hunk 1 above)
Comment (Richard-Mathie) [reply .travis.yml:31]: https://packaging.python.org/distributing/#packaging-your-project

says we should do sdist and wheel

Though it should be a universal wheel.

Comment (Richard-Mathie) [reply .travis.yml:31]: These guys do it in the setuptools setup config

http://www.dougalmatthews.com/2016/Sep/01/automate-publishing-to-pypi-with-pbr-and-travis/

**Comment by thegeorgeous** [comment]
Closing this in favour of #35

**Pull request closed without merging**